common/lib/xmodule/xmodule/mako_module.py

```diff
@@ -39,6 +40,14 @@ def get_context(self):
             'editable_metadata_fields': self.editable_metadata_fields
         }
 
+    def studio_view(self, context):
+        fragment = Fragment(
+            self.system.render_template(self.mako_template, self.get_context())
```

Comment:
Should it then be suffixed with `Mixin`? On its own as a Python class, it doesn't work due to accessing attributes it doesn't itself have - perhaps that suffix would make that clear.